### PR TITLE
Feat/weather api

### LINKED ISSUE #22: Add weather api
## Value Statement
In order to have a better travel experience, the user should be able to check the weather in the desired destination.

## Description
The weather is shown in the saved trip.

## Acceptance Criteria
- [x] fetch correct data from the weather api
- [x] get icon for better UX

## Tasks 
- [x] fetch correct data from the weather api
- [x] get icon for better UX
- [x] adjust css

## Size
small

### PR DESCRIPTION
**Pull request by yvlange**
In this PR I fetched the weather api. Now the user can see the current weather of the destination he wants to travel to.
Please review and check for any errors! 
Thank you!🤓 

_(PS: A small bug I noticed is that once you add a trip, this trip only shows once you reload the page. Need to check on that)_

### CODE REVIEW

File: src/components/AddForm.js [hunk 1]
```diff
@@ -4,14 +4,16 @@ import UploadPhoto from "./UploadPhoto";
...
 
 function AddForm() {
   const [destinationInput, setDestinationInput] = useState("");
-  const [datesInput, setDatesInput] = useState(new Date());
+  const [datesInput, setDatesInput] = useState("");
   const [activitiesInput, setActivitiesInput] = useState("");
   const [restaurantsInput, setRestaurantsInput] = useState("");
   const [notesInput, setNotesInput] = useState("");
   const [imageUpload, setImageUpload] = useState("");
+  const history = useHistory();
```
Comment (hugh-burgess) [inline src/components/AddForm.js:16]: i think you can say `let` no?

File: src/components/AddForm.js [hunk 2]
```diff
@@ -95,10 +91,14 @@ function AddForm() {
           }}
         />
         <div className="form__buttons">
+          {/* <Link to="/trips"> */}
```
Comment (hugh-burgess) [inline src/components/AddForm.js:94]: i spy something commented!

File: src/components/SavedTrip.js [hunk 3]
```diff
@@ -43,11 +43,10 @@ function SavedTrip() {
       <TripDetails name="restaurants" text={singleTrip.restaurants} />
       <TripDetails name="notes" text={singleTrip.notes} />
       <Image
+        className="uploadedImage"
         cloudName="dyjecx1wm"
         publicId={singleTrip.photo}
         secure="true"
```
Comment (hugh-burgess) [inline src/components/SavedTrip.js:49]: what does this do??

File: src/styles/Weather.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
+.weatherContainer {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+  background: rgba(255, 255, 255, 0.05);
+  box-shadow: 0 25px 45px rgba(0, 0, 0, 0.2);
```
Comment (hugh-burgess) [inline src/styles/Weather.css:9]: you can make these into custom variables! like `background: var(--color)` or `box-shadow: var(--shadow)`

**Review by hugh-burgess — CHANGES_REQUESTED** [review]
LTMG! Just some small points

**Review by FriederikeM — APPROVED** [review]
Looks great. I love the weather info!
Unfortunately I don't have an idea about the trip only showing up on second render 🙈

**Pull request merged**